### PR TITLE
Fix classrefs

### PR DESCRIPTION
**Pull request by thekid**
Hey @kiesel this looks good - I'd like to merge it if that's OK for you:)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # This provides an initial configuration to
 # test xp-framework on http://travis-ci.org/
 
+sudo: false
```
Comment (thekid) [inline .travis.yml]: Not sure what this does here though:)

**Comment by kiesel** [comment]
I haven't had the time to fully go through all the classes, but I guess it's good to merge anyways - other findings will be independent ones.... Good to merge.

**Comment by thekid** [comment]
Thanks!

File: .travis.yml (hunk 1 above)
Comment (kiesel) [inline .travis.yml]: This disables `sudo` permissions on the travis buildbox (which the build doesn't use currently), therefore enables use of containerized boxes (using Docker) and thus results in faster build times.

**Pull request merged**